OpenEdX/Info.plist

```diff
@@ -31,5 +31,9 @@
 	<array/>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<true/>
+	<key>NSCalendarsUsageDescription</key>
+	<string>edX would like to use your calendar list to subscribe to your personalized edX calendar for this course.</string>
+	<key>NSCalendarsFullAccessUsageDescription</key>
+	<string>edX would like to use your calendar list to subscribe to your personalized edX calendar for this course.</string>
```

Comment:
Could you please not hardcode the name of the edX app?
Ideally, we should use `platformName` or something similar from the `config.yaml`.